docs/user-guide/gpudeduplication.rst

```diff
@@ -5,164 +5,367 @@
+    )
 
-This will create a new field named :code:`my_id` within each json document which will have the form "doc_prefix-000001".
-If the dataset already has a unique ID this step can be skipped.
+    duplicate_docs = ExactDups(dataset)
 
-**Note**: Fuzzy deduplication only works with numeric ID's or the specific ID format generated by the :code:`add_id` script. If the
-dataset does not contain ID's in this format it's recommended to convert to an integer based ID or ID created by the :code:`add_id` script.
+.. tip::
+  A more comprehensive example, including how to remove documents from a corpus using the list of
+  duplicate IDs generated from exact deduplication for the above can be found in `examples/exact_deduplication.py <https://github.com/NVIDIA/NeMo-Curator/blob/main/examples/exact_deduplication.py>`_.
 
-Once a unique ID has been added to each document, users can proceed with exact and fuzzy deduplication which roughly require the following
-steps (all scripts are included in the :code:`nemo_curator/scripts/` subdirectory):
+""""""""""""
+CLI Utility
+""""""""""""
+Assuming that a unique ID has been added to each document, users can proceed with finding exact duplicates
+as follows:
 
-* Exact dedup
+* Find Exact Duplicates
     1. Input: Data directories
-    2. Output: _exact_duplicates.parquet. List of exact duplicates and the document hash.
+    2. Output: ``_exact_duplicates.parquet``. List of exact duplicates and the document hash.
 
-* Fuzzy Dedup
+.. code-block:: bash
 
-  1. Compute Minhashes
-    - Input: Data Directories
-    - Output: minhashes.parquet for each data dir.
-    - Example call:
+        # same as `python nemo_curator/scripts/find_exact_duplicates.py`
+         gpu_exact_dups \
+           --input-data-dirs /path/to/jsonl/dir1 /path/to/jsonl/dir2 \
+           --output-dir /path/to/output_dir \
+           --input-json-text-field text_column_name \
+           --input-json-id-field id_column_name \
+           --log-dir ./
+           # --scheduler-file /path/to/file.json
 
-         .. code-block:: bash
+All CLI scripts are included in the :code:`nemo_curator/scripts/` subdirectory.
 
-                 # same as `python compute_minhashes.py`
-                 gpu_compute_minhashes \
-                   --input-data-dirs /path/to/jsonl/dir1 /path/to/jsonl/dir2 \
-                   --output-minhash-dir /path/to/output_minhashes \
-                   --input-json-text-field text_column_name \
-                   --input-json-id-field id_column_name \
-                   --minhash-length number_of_hashes \
-                   --char-ngram char_ngram_size \
-                   --hash-bytes 4(or 8 byte hashes) \
-                   --seed 42 \
-                   --log-dir ./
-                   # --scheduler-file /path/to/file.json
+.. caution::
+    The CLI utilities are limited to Jsonl datasets and only work with GPU based backends.
+    For different dataset formats or backeds use the :ref:`exactdup_pyapi`.
```

Comment:
```suggestion
    For different dataset formats or backends use the :ref:`exactdup_pyapi`.
```
Also, where is `exactdup_pyapi` pointing to?

Reply:
It doesn't render on GitHub, but the final docs build will redirect to Python API section above: https://github.com/NVIDIA/NeMo-Curator/pull/258/files#diff-3d426f2a1cbd402015b9a962eea5aba3ccb0d60041076e42958705f64a7f8ca0R38